go/tasks/plugins/k8s/sagemaker/constants.go

```diff
@@ -0,0 +1,41 @@
+package sagemaker
+
+import commonv1 "github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common"
+
+const (
+	pluginID          = "aws_sagemaker_hpo"
+	sagemakerTaskType = "aws_sagemaker_hpo"
+)
+
+
+const (
```

Comment:
lets keep the constants in the file (hpo) where we use it

Reply:
should these be enumer?

Reply:
These contants need have to  this type `commonv1.HyperParameterTuningJobStrategyType` tho, which is just a  redefinition of string.

In `github.com/aws/amazon-sagemaker-operator-for-k8s/api/v1/common/sagemaker_api.go`
```Go
type HyperParameterTuningJobStrategyType string
```